easybuild/easyconfigs/b/biom-format/biom-format-2.1.10-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     Revised BSD
+#
+# Notes::
+##
+
+easyblock = 'PythonPackage'
+
+name = 'biom-format'
+version = '2.1.10'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://biom-format.org'
+description = """
+The BIOM file format (canonically pronounced biome) is designed to be
+ a general-use format for representing biological sample by observation
+ contingency tables. BIOM is a recognized standard for the Earth Microbiome
+ Project and is a Genomics Standards Consortium supported project.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://files.pythonhosted.org/packages/2b/8f/\
```

Comment:
@robqiao are you sure you need that unusual `source_urls`? It fails with a control character error when I try a test build. For `PythonPackage` do you actually need `source_urls` to be defined at all?

Reply:
@verdurin you are right, `curl` failed the default pypi path but `eb` got a hold of the gzip. Hmm ...